### DISCUSSION TITLE
Question about estimates.C

### DISCUSSION THREAD

**Discussion by bioBaddie** [flatironinstitute/CaImAn#1327] [Q&A] [answered]
I recently used CNMF-E to analyze my 1-photon calcium imaging data. I was very happy with the results, but when I presented the data to my lab mates, they had some questions about the denoised traces that I didn't know the answers to. 

1) Could someone explain the math behind the denoised traces? (i.e. what is being done to the data to make the baseline essentially zero?) These transients look so clean, I was wondering what is being done to make them look like this? 
2) I have heard that the values on the denoised traces are sort of arbitrary. My lab mates were asking me if these values corresponded to pixel intensities? I told them no, but didn't have a great explanation. 

Could someone help me understand the denoised traces? Thank you :)

**Comment by kushalkolar** [comment, marked as answer]
> These transients look so clean, I was wondering what is being done to make them look like this?

The tl;dr without getting into too much detail is, an autoregressive model is used to infer the calcium dynamics "based on what we know calcium dynamics are supposed to look like", the oasis paper goes in detail on this: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005423 

> I have heard that the values on the denoised traces are sort of arbitrary. My lab mates were asking me if these values corresponded to pixel intensities? I told them no, but didn't have a great explanation.

They are not arbitrary, CNMF minimizes `|| Y - AC - bf ||`, `C` is the demixed (from neighboring neuron) and deconvolved activity. You can reconstruct a denoised movie if you do `AC + bf`.

If you're a beginner I would recommend watching Andrea's open neuroscience talk, see: https://github.com/flatironinstitute/CaImAn?tab=readme-ov-file#videos

**Comment by bioBaddie** [comment]
Thanks for the input - this is helpful! I guess the only reason I said I had seen somewhere that the units for estimates.C are arbitrary is because this was discussed in the response to question #1273.
